packages/vc-verification/test/issuer-verification-test.ts

```diff
@@ -726,5 +726,175 @@ function testSuite() {
         )
         .and.be.an.instanceOf(Error);
     });
+
+    it('verifies issuer with RoleEIP191Jwt, where expiration timestamp is valid', async () => {
+      const adminJWT = new JWT(adminKeys);
+      const claimAdmin = {
+        claimData: { fields: {}, claimType: adminRole, claimTypeVersion: 1 },
+        iss: adminDid,
+        exp: Date.now() + 10000,
+        credentialStatus: {
+          id: 'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+          type: 'StatusList2021Entry',
+          statusPurpose: 'revocation',
+          statusListIndex: '0',
+          statusListCredential:
+            'https://identitycache-dev.energyweb.org/v1/status-list/urn:uuid:4fb4e120-a566-499c-85fb-47bb5abd3d6b',
+        },
+        signer: adminDid,
+      };
+      let token: string = '';
+      let ipfsCIDAdmin: string = 'ipfsUrl';
+      if (admin.privateKey) {
+        token = await adminJWT.sign(claimAdmin);
+        if (token) {
```

Comment:
What is the case when `sign` returns nothing?

Reply:
Will remove the check, it was because of the parent check for privateKey.